modules/persistent-storage-local-install.adoc

```diff
@@ -13,27 +13,24 @@ The Local Storage Operator is not installed in {product-title} by default. Use t
 
 .Procedure
 
-. Create the `local-storage` project:
+. Create the `openshift-local-storage` project:
 +
 [source,terminal]
 ----
-$ oc new-project local-storage
+$ oc new-project openshift-local-storage
 ----
 
-. Optional: Allow local storage creation on master and infrastructure nodes.
+. Optional: Allow local storage creation on infrastructure nodes.
 +
-You might want to use the Local Storage Operator to create volumes on master and infrastructure nodes, and not just worker nodes, to support components such as logging and monitoring.
+You might want to use the Local Storage Operator to create volumes on infrastructure nodes in support of components such as logging and monitoring.
 +
-To allow local storage creation on master and infrastructure nodes, add a toleration to the DaemonSet by entering the following commands:
+You must adjust the default node selector so that the Local Storage Operator includes the infrastructure nodes, and not just worker nodes.
 +
-[source,terminal]
-----
-$ oc patch ds local-storage-local-diskmaker -n local-storage -p '{"spec": {"template": {"spec": {"tolerations":[{"operator": "Exists"}]}}}}'
-----
+To block the Local Storage Operator from inheriting the cluster-wide default selector, enter the following command:
 +
 [source,terminal]
 ----
-$ oc patch ds local-storage-local-provisioner -n local-storage -p '{"spec": {"template": {"spec": {"tolerations":[{"operator": "Exists"}]}}}}'
+$ oc annotate project local-storage openshift.io/node-selector=''
```

Comment:
@bobfuru Should this command also be `openshift-local-storage` instead of `local-storage`?